crates/evm-tracing-client/src/utils.rs

```diff
@@ -0,0 +1,166 @@
+//! Utils implementations.
+
+/// Check whether it's a known opcode or not. In case it's a known one,
+/// return the name of the opcode then.
+pub fn is_known_opcode(opcode: &evm::Opcode) -> Option<&'static str> {
+    Some(match opcode.0 {
```

Comment:
They made the internal byte of the structure public, but I think it's better to access it using the `as_u8` method that will be inlined anyway. By the way, regarding Mozgiii's question about the situation where the structure size grows, they will have to return an `Option` in `as_u8` or find another way out of the situation, so this can serve as an additional safeguard for that case.

Reply:
https://github.com/humanode-network/humanode/pull/1642/commits/e661328c9ef41f3877bab0b38871478203b346a0